pkg/trace/stats/concentrator_test.go

```diff
@@ -245,8 +328,9 @@ func TestConcentratorStatsCounts(t *testing.T) {
 		testSpan(6, 0, 24, 0, "A1", "resource2", 0),
 	}
 
-	expectedCountValByKeyByTime := make(map[int64]map[string]int64)
-	expectedCountValByKeyByTime[alignedNow-2*testBucketInterval] = map[string]int64{
+	expectedCountValByKeyByTime := make(map[int64]map[string]float64)
+	// 2-bucket old flush
+	expectedCountValByKeyByTime[alignedNow-2*testBucketInterval] = map[string]float64{
```

Comment:
The new `expected` map that `countValsEq` expects is of type `map[string]float64`.